UIkitDZ/VC/PizzaIngridients.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  PizzaIngridients.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 01.07.2021.
+//
+
+import UIKit
+
+class PizzaIngridients: UIViewController {
+    
+    var pizzaImageStr = String()
```

Comment:
```suggestion
  private  var pizzaImageStr = String()
```
и все что ниже private